frontend/webpack.production.config.js

```diff
@@ -65,6 +65,7 @@ const config = {
   },
   plugins: [
     new Webpack.DefinePlugin({
+      __LOGOUT_TEST_URL__: JSON.stringify('https://logontest.gov.bc.ca/clp-cgi/logoff.cgi'),
```

Comment:
Is there a reason to have the test URL in production config?

I think is might be better externalized to a build config file to avoid exposing this in bundle.js?

Reply:
Unfortunately, this seems to be a limitation with our deployment process. Initially, webpack.config.js was intended to be for development and webpack.production.config.js for test and prod. But it seems that dev is using production configuration as well. Kuan tried changing the settings so that dev uses webpack.config.js, but it didn't work.